src/main/java/de/zalando/ep/zalenium/servlet/GridStatusServlet.java

```diff
@@ -0,0 +1,58 @@
+package de.zalando.ep.zalenium.servlet;
+
+import static com.google.common.net.MediaType.JSON_UTF_8;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.zalando.ep.zalenium.registry.ZaleniumRegistry;
+import de.zalando.ep.zalenium.util.ZaleniumConfiguration;
+import static java.net.HttpURLConnection.HTTP_OK;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import org.openqa.grid.web.servlet.RegistryBasedServlet;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Optional;
+
+public class GridStatusServlet extends RegistryBasedServlet {
+
+    private static final long serialVersionUID = 1L;
+
+    @SuppressWarnings("unused")
+    public GridStatusServlet() {
+        this(null);
+    }
+
+    public GridStatusServlet(ZaleniumRegistry registry) {
+        super(registry);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        if (getRegistry() == null) {
+            throw new RuntimeException("Registry is null");
+        }
+
+        int allProxiesCount = Optional.ofNullable(getRegistry().getAllProxies().size()).orElse(0);
+
+        GridStatus status = ZaleniumConfiguration.getDesiredContainersOnStartup() <= allProxiesCount ? GridStatus.FULL_CAPACITY
+                : allProxiesCount > 0 ? GridStatus.AVAILABLE
+                : GridStatus.UNAVAILABLE;
+
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        byte[] statusReturn = gson.toJson(status).getBytes(UTF_8);
+        response.setStatus(HTTP_OK);
```

Comment:
Should it not reply with SERVICE_UNAVAILABLE http status in case the grid is unavailable ?

Reply:
I think my documentation is not really correct, the status that I consider unavailable on this servlet is actually a grid available but without any attached node.
If the GRID is not started, this servlet will not respond with a 200 return code.
I will update my documentation to be more precise.
Is it ok for you?

Reply:
https://github.com/zalando/zalenium/pull/1003/commits/fcb44328cc616424f68623c8bfa3782b460f9a6b